src/linalg/mod.rs

```diff
@@ -265,7 +266,7 @@ pub trait BaseVector<T: RealNumber>: Clone + Debug {
             sum += xi * xi;
         }
         mu /= div;
-        sum / div - mu * mu
+        sum / div - mu.powi(2)
```

Comment:
I am wondering what is faster in Rust, mu * mu or mu.powi(2)? I always thought that multiplication is faster but that might not be true in Rust. This might be a small thing, but if repeated many times the difference will accumulate to a significant value.

Reply:
nm, I found my answer here: https://stackoverflow.com/questions/38799724/is-there-a-performance-difference-between-a-squared-function-and-directly-callin